core/string/ustring.h

```diff
@@ -531,21 +531,13 @@ struct FileNoCaseComparator {
 };
 
 template <typename L, typename R>
-_FORCE_INLINE_ bool is_str_less(const L *l_ptr, const R *r_ptr) {
+_FORCE_INLINE_ int64_t str_compare(const L *l_ptr, const R *r_ptr) {
```

Comment:
If you're renaming anyway, it would be a good opportunity to allude to the fact that you're doing a `char32_t` comparison specifically. Maybe call it something like `str32_compare` instead?

Reply:
this is templated, its `char`, `wchar_t`, `char32_t`, `char16_t`, `int`, whatever. it may use `char32_t` inside, but its for whatever cstr type goes in, i think a generic name fits